packages/react-checkbox/src/components/Checkbox/useCheckbox.tsx

```diff
@@ -1,101 +1,101 @@
 import * as React from 'react';
 import {
-  getNativeElementProps,
+  getPartitionedNativeProps,
   resolveShorthand,
   useControllableState,
+  useEventCallback,
   useId,
   useIsomorphicLayoutEffect,
   useMergedRefs,
-  useEventCallback,
 } from '@fluentui/react-utilities';
-import { CheckboxProps, CheckboxSlots, CheckboxState } from './Checkbox.types';
+import { CheckboxProps, CheckboxState } from './Checkbox.types';
 import { Mixed12Regular, Mixed16Regular, Checkmark12Regular, Checkmark16Regular } from './DefaultIcons';
 import { Label, LabelProps } from '@fluentui/react-label';
 
-/**
- * Array of all shorthand properties listed as the keys of CheckboxSlots
- */
-export const checkboxShorthandProps: Array<keyof CheckboxSlots> = ['root', 'indicator', 'input'];
-
 /**
  * Create the state required to render Checkbox.
  *
  * The returned state can be modified with hooks such as useCheckboxStyles,
  * before being passed to renderCheckbox.
  *
  * @param props - props from this instance of Checkbox
- * @param ref - reference to root HTMLElement of Checkbox
+ * @param ref - reference to <input> element of Checkbox
```

Comment:
this needs backticks to avoid trying to render the element
```suggestion
 * @param ref - reference to `<input>` element of Checkbox
```